### ISSUE TITLE
Is this project still Alive ?

### ISSUE THREAD

**Issue by MarouaneMan** [doug-martin/nestjs-query#1538]
All is in the title.

**Comment by TriPSs** [comment]
I'm actively using it and also created multiple PR's, never got an answer on any of them. 

So I don't think so :(

**Comment by smolinari** [comment]
Yeah, and I'd be interested in the maintenance of only one part of the packages, not all of them. So, if others think similarly, taking over the maintenance would be a difficult decision. 

Scott

**Comment by TriPSs** [comment]
@smolinari what parts are you interested in?

**Comment by smolinari** [comment]
@TriPSs - I've worked a little and am still using the Typegoose bit. 

Scott

**Comment by Goopil** [comment]
I'm in a similar position. But use the typeorm adapter.
I've started to retovate things here https://github.com/Goopil/nestjs-query
There is still a lot of work throuth

**Comment by TriPSs** [comment]
@smolinari I do have a fork of this package [here](https://github.com/TriPSs/nestjs-query) that already has some fixes and i'm planning to fix more in it.

It's not yet published some where (I published to private repo), will check if I can make some time soon to publish it somewhere, than maybe the development can continue there.

**Comment by GP4cK** [comment]
@TriPSs I've also made a PR about a month and a half ago. Let me know when it's published publicly ;)

**Comment by TriPSs** [comment]
@GP4cK it currently is :) 

I do have to note that I created a breaking change, still need to document this, but the change is that if the item is not found is used to return null and that now throws an error.

See [this commit](https://github.com/TriPSs/nestjs-query/commit/095396eb466483802cd76c2e9f9b934eb171d17c#diff-aa8426aa038010d7c0488821357a02290ceb70333e71560e76e949159c062d6eL63) for the change.

**Comment by GP4cK** [comment]
I mean the npm packages. Have you published them? I couldn't find them just now...

**Comment by TriPSs** [comment]
Yes:
https://www.npmjs.com/package/@ptc-org/nestjs-query-core
https://www.npmjs.com/package/@ptc-org/nestjs-query-graphql
https://www.npmjs.com/package/@ptc-org/nestjs-query-typeorm
https://www.npmjs.com/package/@ptc-org/nestjs-query-typegoose
https://www.npmjs.com/package/@ptc-org/nestjs-query-mongoose
https://www.npmjs.com/package/@ptc-org/nestjs-query-sequelize

**Comment by meodemsao** [comment]
@TriPSs can you list your bug have been fixed
i fixed 2 bug about enum type and ilike operator, i will pull request to your fork

**Comment by TriPSs** [comment]
@meodemsao Issues/PR's that are merged/released:
- #1504
- #1437, implemented feature to disable filtering and sorting
- #1303
- Fixed `between` filter not working when using custom `allowedComparisons`
- #1495
- #1444
- #1450
- #1447 
- #1445
- https://github.com/TriPSs/nestjs-query/pull/4

**Comment by daniloras** [comment]
Latest version ? .v0.30.0 - 2021-09-30

Is active lib? 
many issues are open

This lib really helps  in development, let's not let this project die, please....

**Comment by smolinari** [comment]
@daniloras - Did you read this thread? @TriPSs  is upkeeping a fork. See here: https://github.com/doug-martin/nestjs-query/issues/1538#issuecomment-1127417301

Scott

**Comment by wowczarczyk** [comment]
It's a bit sad that @doug-martin seemed to be treating this project with care and attention then suddenly completely disappeared. I know it's OS, requires time, etc., but some guidance for the future or designating new maintainers would be nice

**Comment by ruudvanbuul** [comment]
> It's a bit sad that @doug-martin seemed to be treating this project with care and attention then suddenly completely disappeared. I know it's OS, requires time, etc., but some guidance for the future or designating new maintainers would be nice

I honestly find it hard to understand how some of these packages have taken so much work and then they just get abandoned... At least handing over the reigns so others can take over would be nice.

I guess you can always fork but doesn't add to the usability

**Comment by jamesmcglinn** [comment]
> I honestly find it hard to understand how some of these packages have taken so much work and then they just get abandoned... At least handing over the reigns so others can take over would be nice.

Github really doesn't make it easy to hand over the reigns on personal repos. @doug-martin has done incredible work putting this project together, I'm sure he'd keep it up if he could. Really difficult to have to shoulder responsibility for the project when things change and it's no longer a top priority, and there's no way to set up another admin.

**Comment by Diluka** [comment]
@doug-martin thanks for the great project. if you want to leave, can you transfer this project to someone?

**Comment by smolinari** [comment]
It looks like @doug-martin has completely stopped working on code, at least code stored at Github, since July. 

I'd suggest looking at @TriPSs 's fork and supporting his work going forward. 

https://github.com/tripss/nestjs-query 

Scott

**Comment by dvins** [comment]
After seeing over the years what a frequent contributor he has been to the OpenSource community I am concerned for @doug-martin; hope all is well with him.

**Comment by rymanso** [comment]
thank you @TriPSs this is a real saviour :)
